src/components/FindMovie/FindMovie.tsx

```diff
@@ -12,46 +54,77 @@ export const FindMovie: React.FC = () => {
 
           <div className="control">
             <input
+              className={classNames(
+                'input',
+                {
+                  'is-dander': erorr,
+                },
+              )}
               data-cy="titleField"
               type="text"
               id="movie-title"
               placeholder="Enter a title to search"
-              className="input is-dander"
+              value={query}
+              onChange={e => {
+                setQuery(e.target.value);
+                setError(false);
+              }}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {erorr && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
+              className={classNames(
+                'button is-light',
+                {
+                  'is-loading': loading,
+                },
+              )}
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              disabled={!query}
+              onClick={(e) => {
+                e.preventDefault();
+                loadMovie();
+              }}
```

Comment:
good practice do not pass anonymous functions
assign it to variable and use here